resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerConfiguration.java

```diff
@@ -61,4 +62,55 @@ public CircuitBreakerAspect circuitBreakerAspect(CircuitBreakerConfigurationProp
     public EventConsumerRegistry<CircuitBreakerEvent> eventConsumerRegistry() {
         return new DefaultEventConsumerRegistry<>();
     }
+    
+    /**
+     * Initializes the backends configured in the properties.
+     * 
+     * @param circuitBreakerRegistry The circuit breaker registry.
+     * @param eventConsumerRegistry The event consumer registry.
+     */
+    public void initializeBackends(CircuitBreakerRegistry circuitBreakerRegistry,
+    							   EventConsumerRegistry<CircuitBreakerEvent> eventConsumerRegistry) {
+    	
+    	circuitBreakerProperties.getBackends().forEach(
+                (name, properties) -> {
+                    CircuitBreakerConfig circuitBreakerConfig = circuitBreakerProperties.createCircuitBreakerConfig(name);
+                    circuitBreakerRegistry.circuitBreaker(name, circuitBreakerConfig);
+                }
+        );
+    	
+    }
+    
+    /**
+     * Registers the post creation consumer function that registers the consumer events to the circuit breakers.
+     * 
+     * @param circuitBreakerRegistry The circuit breaker registry.
+     * @param eventConsumerRegistry The event consumer registry.
+     */
+    public void registerPostCreationEventConsumer(CircuitBreakerRegistry circuitBreakerRegistry,
+    		EventConsumerRegistry<CircuitBreakerEvent> eventConsumerRegistry) {
+    	eventConsumerRegister = new EventConsumerRegister(eventConsumerRegistry);
+		circuitBreakerRegistry.registerPostCreationConsumer(
+				(circuitBreaker, config) -> eventConsumerRegister.registerEventConsumer(circuitBreaker, config));
+    }
```

Comment:
you can use method reference eventConsumerRegister::registerEventConsumer for better readability